### PR TITLE
GPG-808 Admin can add new address if none

### PR DESCRIPTION
**Pull request by malinaslevoaca1**
See ticket [GPG-808](https://technologyprogramme.atlassian.net/browse/GPG-808) - details about the bug in comments.

Testing:
* Tested locally. Once the employer has an address, the user can add it to their account and report successfully

Risk: None
Documentation: Not required

### CODE REVIEW

**Review by xonde — APPROVED** [review]
Code looks all good, just wanted to clarify, were admins not able to edit the address before? Did it throw an error when attempting to load the page for an organisation without and address?

File: GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationAddressController.cs [hunk 1]
```diff
@@ -202,9 +203,13 @@ private IActionResult CheckChangesAction(ChangeOrganisationAddressViewModel view
...
         {
-            string oldAddressString = organisation.GetLatestAddress().GetAddressString();
+            OrganisationAddress latestAddress = organisation.GetLatestAddress();
+            string oldAddressString = latestAddress?.GetAddressString();
 
-            RetireOldAddress(organisation);
+            if (latestAddress != null)
+            {
+                RetireOldAddress(organisation);
+            }
```
Comment (xonde) [inline GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationAddressController.cs:212]: Not a big deal, but was just wondering if we could just do:
```
string oldAddressString = organisation.GetLatestAddress()?.GetAddressString()

if (oldAddressString != null)
{
RetireOldAddress(organisation)
}
```

That way we don't need the `latestAddress` variable. I could be missing something, but just though I'd check

**Comment by malinaslevoaca1** [comment]
> Code looks all good, just wanted to clarify, were admins not able to edit the address before? Did it throw an error when attempting to load the page for an organisation without and address?

An error was thrown when they tried to change it (NullPointerException)

File: GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationAddressController.cs (hunk 1 above)
Comment (malinaslevoaca1) [reply GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationAddressController.cs]: Nice! 👍

**Review by xonde — APPROVED** [review]
LGTM! 🚀

**Pull request merged**